grakn-graql/src/main/java/ai/grakn/graql/internal/reasoner/cache/LazyQueryCache.java

```diff
@@ -164,7 +159,4 @@ public void reload(){
         this.putAll(newCache);
     }
 
-    public void consume() {
-        this.entries().forEach(entry -> entry.cachedElement().stream().collect(Collectors.toSet()));
-    }
 }
```

Comment:
Could we restore the overrides here as we did for `QueryCache`?

Reply:
Done 👍